File: 2011.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        res = 0

        for operation in operations:
            x, y = operation[0], operation[-1]
            if(x == "+" or y == "+"):
                res += 1
            elif(x == "-" or y == "-"):
                res -= 1
        
        return res

File: 2011.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int res = 0, n;
        for(int i = 0; i < operations.size(); i++) {
            if(operations[i][0] == '+' || operations[i][2] == '+') {
                res += 1;
            } else {
                res -= 1;
            }
        }
        return res;
    }
};